moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -176,16 +221,19 @@ class PlanningComponent
   // The robot_model_ member variable of MoveItCpp class will manually free the joint_model_group_ resources
   const moveit::core::JointModelGroup* joint_model_group_;
 
+  std::mutex plan_mutex_;
```

Comment:
I did a search for this variable but I don't see it used anywhere?

Reply:
That's a leftover, thank you! Initially I thought certain function segments of the planning component are not thread safe. I'll remove it